src/main/java/org/fundacionjala/sfdc/commons/DriverManager.java

```diff
@@ -48,7 +48,7 @@ public static WebDriver getNavigator() {
      * Getter of WebDriverWait object.
      * @return WebDriverWait instance.
      */
-    public static WebDriverWait getWaiter() {
+    public  WebDriverWait getWaiter() {
         if (wait == null) {
             wait = new WebDriverWait(driver, SECONDS_WAIT);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)